invenio/modules/communities/templates/communities/index_base.html

```diff
@@ -30,6 +30,12 @@
 <div class="row">
```

Comment:
communities was kind of already transferred to inveniosoftware/invenio-communities? Perhaps we just remove communities from Invenio instead?